docs/template.md

```diff
@@ -223,6 +223,21 @@ robot template --merge-before \
   --output results/test_template.owl
 ```
 
+ROBOT template data read from separate external file
+
+    robot template --template animals_ext_template.tsv \
+        --ext-template animals_template.tsv \
```

Comment:
I think it's confusing that `--template` uses `animals_ext_template.tsv` while `--ext-template` uses `animals_template.tsv`. The names of these files should be swapped.